infrastructure/scripts/run-smoke-tests.sh

```
@@ -0,0 +1,120 @@
+
+LOCALSTACK_HOST="${LOCALSTACK_HOST:-localhost}"
+LOCALSTACK_PORT="${LOCALSTACK_PORT:-4566}"
+LOCALSTACK_URL="http://${LOCALSTACK_HOST}:${LOCALSTACK_PORT}"
+LOCALSTACK_IMAGE="localstack/localstack:2026.6.0"
+LOCALSTACK_CONTAINER="localstack-smoke"
+
+STARTED_CONTAINER=false
+
+wait_for_localstack() {
+  echo "Waiting for LocalStack to be ready..."
+  for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)